content/docs/configuration/configuration.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Configuration
+category: Configuration
+slug: 6.5.configuration
+hash: "#configuration"
+fullpath: /docs
+---
+
+Audiobookshelf is configured via environment variables.
+You can pass them to your Docker container using `-e VARIABLE=VALUE`
+or set them in `/etc/default/audiobookshelf` if you install audiobookshelf via packages.
+
+Here is a list of all available options:
+
+## Filesystem
+
+- `CONFIG_PATH` (default: `./config`)
+  - Path to the config directory.
+  - It will contain the database (users/books/libraries/settings).
+- `METADATA_PATH` (default: `./metadata`)
+  - Path to the metadata directory.
+  - It will contain cache, streams, covers, downloads, backups and logs.
+
+## External Tools
+
+- `FFMPEG_PATH` (default: `ffmpeg`)
+  - Path to the `ffmpeg` binary.
+  - If no path is set, Audiobookshelf will assume the binary to exist in the system path.
+- `FFPROBE_PATH` (default: `ffprobe`)
+  - Path to the `ffprobe` binary.
+  - If no path is set, Audiobookshelf will assume the binary to exist in the system path.
+- `TONE_PATH` (default: `tone`)
+  - Path to the `tone` binary.
+  - If no path is set, Audiobookshelf will assume the binary to exist in the system path.
+
+## Network
+
+- `HOST` (default: *empty*)
+  - The host Audiobookshelf binds to.
+    Most commonly, this will be `127.0.0.1` if you want the service to listen to localhost only,
+    or empty if you want to listen to all interfaces (both IPv4 and IPv6).
+- `PORT` (default: `3333`)
+  - The TCP port Audiobookshelf will listen on.
+
+## Security
+
+- `TOKEN_SECRET`
+  - Secret used for generating the JSON Web Tokens.
+  - If none is provided, a secure random token is generated automatically.
+    That will usually be sufficient.
+
+## Other
+
+- `AUDIOBOOKSHELF_GID`
+  - Numeric group identifier to which Audiobookshelf will assign some of its managed files.
+  - If not specified, Audiobookshelf will simply write files using the default permissions of the user it runs as.
+  - To use this, you must specify both `uid` and `gid`.
+- `AUDIOBOOKSHELF_UID`
+  - Numeric user identifier to which Audiobookshelf will assign some of its managed files.
+  - If not specified, Audiobookshelf will simply write files using the default permissions of the user it runs as.
+  - To use this, you must specify both `uid` and `gid`.
+- `NODE_ENV`
+  - The environment Audiobookshelf is run in, influencing e.g. additional logging.
+  - Can be `development` or `production`.
```

Comment:
```suggestion
```

I don't think we should document this because if `development` is used then the client will require a server at `localhost:3333`. It will probably just confuse people trying to get debug logs.

Reply:
I don't care much either way, but I would tend to keeping this around but document that people shouldn't mess with it if they don't know what they are doing.  Hiding options will confuse people who are trying to use them, for example because they want to do some development.

Reply:
I'm also not sure this would work. It could be that in Nuxt the env is set at build time in which case passing this in would only partially implement the dev mode. This is in `client/nuxt.config.js`

Reply:
Then, let's remove this. I can take care of that tomorrow.